.changesets/fix_bryn_datadog_agent_sampling.md

```diff
@@ -0,0 +1,34 @@
+### Add `preview_datadog_agent_sampling` ([PR #6017](https://github.com/apollographql/router/pull/6017))
+
+The sampler option in the `telemetry.exporters.tracing.common.sampler` is not datadog aware.
+
+To get accurate APM metrics all spans must be sent to the datadog agent with a `psr` or `sampling.priority` attribute set appropriately to record the sampling decision.
+
+`preview_datadog_agent_sampling` option in the router.yaml enables this behavior and should be used when exporting to the datadog agent via OTLP or datadog native. 
+
+```yaml
+telemetry:
+  exporters:
+    tracing:
+      common:
+        # Only 10 percent of spans will be forwarded from the Datadog agent to Datadog. Experiment to find a value that is good for you!
+        sampler: 0.1
+        # Send all spans to the Datadog agent. 
+        preview_datadog_agent_sampling: true
+
+        
+```
+
+By using these options, you can decrease your Datadog bill as you will only be sending a percentage of spans from the Datadog agent to datadog. 
```

Comment:
```suggestion
Using these options can decrease your Datadog bill, because you will be sending only a percentage of spans from the Datadog Agent to Datadog. 
```